### PR TITLE
Save settings to local storage

### PR DESCRIPTION
**Pull request by webhead3893**

### CODE REVIEW

**Comment by webhead3893** [comment]
Settings completed so far:
Size of info panels
Real time/Game time switch

File: components/SplitsDisplayTab.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
         Graphs' Y axis starts at zero
       </loading-switch>
       <loading-switch v-model="graphCurrentAttemptHline" class="mb-2">
-        Display current attempt times' as an horizontal line
+        Display current attempt times as a horizontal line
```
Comment (Webcretaire) [inline components/SplitsDisplayTab.vue:12]: It seems I was drunk or very tired when I wrote that but what I actually meant was this
```suggestion
        Display current attempt's times as a horizontal line
```

**Comment by Webcretaire** [comment]
Looks good to me, thanks for your contribution as usual. PR is still called "WIP:...." though so if you're done with it remove the "WIP:" prefix and I'll merge it

**Comment by webhead3893** [comment]
Oops, let me fix that

**Pull request merged**